### PR TITLE
Issue #18052: Query total physical memory based on MBean attribute availability

### PR DESCRIPTION
**Pull request by kgibm**
Avoid relying on checking JVM vendor and instead check the availability of the MBean attribute directly

### CODE REVIEW

**Comment by kgibm** [comment]
fixes #18052

**Comment by kgibm** [comment]
#build

**Comment by kgibm** [comment]
#build

**Comment by kgibm** [comment]
Two build errors in IHS that are unrelated. I also confirmed that whether running with J9 or HotSpot, no FFDC is produced.

File: dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/MemoryInformation.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 IBM Corporation and others.
+ * Copyright (c) 2021 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/MemoryInformation.java]: Should list both dates
```suggestion
 * Copyright (c) 2018, 2021 IBM Corporation and others.
```
Don't really need a new PB to fix that though

**Review by tevans78 — APPROVED** [review]
Approved so long as the copyright date is fixed

File: dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/MemoryInformation.java (hunk 1 above)
Comment (kgibm) [reply dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/MemoryInformation.java]: Done, thanks

**Comment by kgibm** [comment]
#run-libby-bot

**Pull request merged**